tools/appliance/systemvmtemplate/scripts/configure_grub.sh

```diff
@@ -20,6 +20,8 @@ set -e
 set -x
 
 function configure_grub() {
+  # Remove old kernel
+  apt-get remove -y --purge linux-image-amd64 linux-image-4.19.0-16-amd64 || true
```

Comment:
@rhtyd two potential issues in my opinion
(1) the versions are different in debian releases (debian 10.9.0: kernel 4.19.0.16.debian 10.8.0: kernel 4.19.0.14,etc)
(2) might there be more than 2 versions ? for example, one new kernel by `apt-get -q -y dist-upgrade`, another new kernel by `apt-get -q -y dist-upgrade -t buster-backports` ?

Reply:
I see, we can explore some general way of removing kernels that are not in use.